lib/node_modules/@stdlib/repl/README.md

```diff
@@ -176,6 +199,126 @@ repl.clearUserDocs();
 repl.close();
 ```
 
+#### Repl.prototype.themes()
+
+Returns all available themes in the syntax-highlighter.
+
+```javascript
+var debug = require( '@stdlib/streams/node/debug-sink' );
+
+// Create a new REPL:
+var repl = new REPL({
+    'output': debug()
+});
+
+// ...
+
+// Fetch all available themes:
+repl.themes();
+
+// ...
+
+// Close the REPL:
+repl.close();
+```
+
+#### Repl.prototype.getTheme()
+
+Returns a theme's color palette from the syntax-highlighter.
+
+```javascript
+var debug = require( '@stdlib/streams/node/debug-sink' );
+
+// Create a new REPL:
+var repl = new REPL({
+    'output': debug()
+});
+
+// ...
+
+// Fetch all available themes:
+repl.themes();
+
+// ...
+
+// Close the REPL:
+repl.close();
+```
+
+#### Repl.prototype.addTheme()
+
+Adds a color theme to the syntax-highlighter.
+
+```javascript
+var debug = require( '@stdlib/streams/node/debug-sink' );
+
+// Create a new REPL:
+var repl = new REPL({
+    'output': debug()
+});
+
+// ...
+
+// Add a user-defined theme:
+repl.addTheme( 'myTheme', {
+    'keyword': 'red',
+    'variable': 'green'
+
+    // ...
+});
+
+// ...
+
+// Close the REPL:
+repl.close();
+```
+
+#### Repl.prototype.deleteTheme()
+
+Deletes a specified theme from the syntax-highlighter.
+
+```javascript
+var debug = require( '@stdlib/streams/node/debug-sink' );
+
+// Create a new REPL:
+var repl = new REPL({
+    'output': debug()
+});
+
+// ...
+
+// Delete an existing theme:
+repl.deleteTheme( 'myTheme' );
+
+// ...
+
+// Close the REPL:
+repl.close();
+```
+
+#### Repl.prototype.renameTheme()
+
+Renames a specified theme in the syntax-highlighter.
+
+```javascript
+var debug = require( '@stdlib/streams/node/debug-sink' );
+
+// Create a new REPL:
+var repl = new REPL({
+    'output': debug()
+});
+
+// ...
+
+// Rename an existing theme:
+repl.renameTheme( 'myTheme', 'yourTheme' );
+
+// ...
+
+// Close the REPL:
+repl.close();
+```
+
```

Comment:
You also need to document REPL commands. See https://github.com/stdlib-js/stdlib/tree/96e896a39be08912b2e06dfb6b671ec13d042412/lib/node_modules/%40stdlib/repl#commands.